src/main/java/core/basesyntax/Main.java

```diff
@@ -0,0 +1,15 @@
+package core.basesyntax;
+
+public class Main {
+    public static void main(String[] args) {
+        FiguresSupplier figureSupplier = new FiguresSupplier();
```

Comment:
The class name should be 'FigureSupplier' instead of 'FiguresSupplier' to match the task description and maintain consistency.